src/components/header-checkout/header-checkout.hbs

```diff
@@ -0,0 +1,181 @@
+<a class="skip-link" id="skip-to-main" href="#skip-to-nav" tabindex="0">Skip navigation menu</a>
```

Comment:
I've removed what is needed, and appended the main class names with `-checkout`.